### ISSUE TITLE
[BUG] Auth doesn't work even i added users to appConfig.auth

### ISSUE THREAD

**Issue by npty** [Lissy93/dashy#1311]
### Environment

Self-Hosted (Docker)

### System

Debian 12

### Version

2.1.1

### Describe the problem

I have configured the auth object like the following

```
appConfig:
  auth:
    enableGuestAccess: false
    users:
    - user: <redact>
      hash: <redact>
      type: admin
```

The problem is anyone can still access to my homepage. I've tried restarted the server, but it doesn't work.

### Please tick the boxes

- [X] You have explained the issue clearly, and included all relevant info
- [X] You are using a [supported](https://github.com/Lissy93/dashy/blob/master/.github/SECURITY.md#supported-versions) version of Dashy
- [X] You've checked that this [issue hasn't already been raised](https://github.com/Lissy93/dashy/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://github.com/Lissy93/dashy/tree/master/docs#readme) and [troubleshooting](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#troubleshooting) guide ![important](https://img.shields.io/badge/Important!-F6094E)
- [X] You agree to the [code of conduct](https://github.com/Lissy93/dashy/blob/master/.github/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct)

**Comment by dirigoIT** [comment]
Many reports of this issue. 
We have this issue also. 
Is this being worked on? Seems to me this is a high priority bug.

**Comment by 7Adrian** [comment]
I've got this issue too (version: 2.1.1). 

Problem:
1. Open page in browser (doesn't matter if I put http://ip or http://ip/login)
2. Main page appear (expected login page)
3. There is no login section*
4. Just only after trying to change view to any option login page appear (this open http://ip/login, but that time correctly)
5. When paste login and password and clicking login button I am logged correctly as expected, but when login page appear when I refresh page I am again at main page (http://ip)

* Login section show in when I correctly logged in.

**Comment by DSI-dot-Guru** [comment]
Any word on this?

**Comment by CrazyWolf13** [comment]
@Lissy93 Think this is a duplicate and could be closed in favor of #990  ?
